### DISCUSSION TITLE
提供低代码平台支持

### DISCUSSION THREAD

**Discussion by lesliechueng** [taowen/awesome-lowcode#119] [General]
提供基于springcloud+vue架构的低代码平台，源码交付，功能强大，包含复杂表单、流程、报表、BI功能，价格亲民，感兴趣扫微信咨询。
![a6fc555f00c7df7942bc0432c330d71](https://user-images.githubusercontent.com/13750154/222945771-7be6e737-ded3-4b82-bcbe-2fece3ea93c5.jpg)

**Comment by LowCoderGitHub** [comment]
搞了个低代码行业的交流平台，大佬有没有兴趣来新平台玩玩
https://www.lowcoder.com.cn/
